electron-app/src/components/settings/api-keys/ExchangeSettings.vue

```diff
@@ -2,8 +2,20 @@
   <v-row class="exchange-settings">
     <v-col>
       <v-card>
-        <v-card-title>Exchange Settings</v-card-title>
+        <v-card-title>Exchanges</v-card-title>
         <v-card-text>
+          <p>
+            Rotki can connect to supported exchanges and automatically obtain
+            your trades and balances in that exchange. Each exchange may require
+            a different set of credentials for the connection, so make sure that
```

Comment:
What do you want to say by:
"Each exchange may require a different set of credentials for the connection, so make sure that everything is provided"?

Reply:
Some require more than just API Key + Secret (well it's only one right now), but perhaps it's redundant if we're linking to the guide.